### ISSUE TITLE
WALL-E: I cannot capture scene and RTX Remix seems to not work?

### ISSUE THREAD

**Issue by Virjoinga** [NVIDIAGameWorks/rtx-remix#290]
**Describe the bug**
When pressing "capture" nothing happens. Also from videos I have seen, usually the scene changes slightly (lighting wise) but not here.
remix-0.3.0
RTX 3080
546.33
Latest version and is on Steam.

**Attach files!**
[NvRemixBridge.log](https://github.com/NVIDIAGameWorks/rtx-remix/files/13868679/NvRemixBridge.log)
[d3d9.log](https://github.com/NVIDIAGameWorks/rtx-remix/files/13868680/d3d9.log)
[WALL-E_d3d9.log](https://github.com/NVIDIAGameWorks/rtx-remix/files/13868681/WALL-E_d3d9.log)

**To Reproduce**
Load into any level and attempt to click capture scene.

**Expected behavior**
I expect RTX Remix to capture the scene.

**Comment by NV-LL** [comment]
Hello! Will you please test this on the latest release and let us know if the issue persists? Thank you!

**Comment by Virjoinga** [comment]
> Hello! Will you please test this on the latest release and let us know if the issue persists? Thank you!

Just tested it with the latest release (0.5.0) and it still does not work.
[NvRemixBridge.log](https://github.com/NVIDIAGameWorks/rtx-remix/files/15258126/NvRemixBridge.log)
[d3d9.log](https://github.com/NVIDIAGameWorks/rtx-remix/files/15258128/d3d9.log)
[WALL-E_d3d9.log](https://github.com/NVIDIAGameWorks/rtx-remix/files/15258130/WALL-E_d3d9.log)

**Comment by PhialsBasement** [comment]
Has this ever  been fixed?

**Comment by Virjoinga** [comment]
> Has this ever  been fixed?

No, I also tried it on the latest release too (via Omniverse). If you need a log, I'll send it.

**Comment by NV-LL** [comment]
Hi @Virjoinga - yes, please do send along that most recent log. Thank you!

**Comment by Virjoinga** [comment]
> Hi @Virjoinga - yes, please do send along that most recent log. Thank you!

Sure, here they are:
[NvRemixBridge.log](https://github.com/user-attachments/files/17001026/NvRemixBridge.log)
[d3d9.log](https://github.com/user-attachments/files/17001027/d3d9.log)
[WALL-E_d3d9.log](https://github.com/user-attachments/files/17001028/WALL-E_d3d9.log)

**Comment by NV-LL** [comment]
Thanks for the updated info! We've opened REMIX-3554 for tracking.
